news/2019-05-10-spring-2019-fanart-contest.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "Spring 2019 Fanart Contest"
+date: 2019-05-10 12:00:00 +0000
+---
+
+Better late than never! Spring might be on the way out, but talent is *always* in season, so whip out your drawing implement of choice and draw any of the osu! mascots enjoying Spring for fun and prizes!
+
+[![](https://assets.ppy.sh/contests/78/header.jpg)](https://osu.ppy.sh/community/contests/78)
+
+If you find yourself a person of artistic interest, this should be right up your alley! The artists among us have **14 days** to take a whirl at drawing any **spring-themed rendition of the osu! official or community mascots**.
```

Comment:
I would prefer the adjectives for mascots be swapped in order, like "official or community-sourced osu! mascots".